meson.build

```diff
@@ -35,6 +35,9 @@ endif
 # pthreads is not a real dependency
 pthreads = dependency('threads')
 
+# declare option to link against libm
+math = declare_dependency(link_args: ['-lm'])
```

Comment:
I think it's better to use:
```
cc = meson.get_compiler('c')
math = cc.find_library('m')
```

Reply:
okay, thank you very much, i will change this.